model/security/patch_tenant/operations.smithy

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+use opensearch.openapi#vendorExtensions
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/2.7/security/access-control/api/#patch-tenant"
+)
+
+@vendorExtensions(
+    "x-operation-group": "security.patch_tenant",
+    "x-version-added": "1.0",
+)
+@idempotent
+@suppress(["HttpUriConflict"])
+@http(method: "PATCH", uri: "/_plugins/_security/api/tenants/{tenant}")
+@documentation("Add, delete, or modify a single tenant.")
+operation PatchTenant {
+    input: PatchTenant_Input,
+    output: PatchTenant_Output
+}
+
+@vendorExtensions(
+    "x-operation-group": "security.patch_tenants",
+    "x-version-added": "1.0",
+)
+@idempotent
+@suppress(["HttpUriConflict"])
+@http(method: "PATCH", uri: "/_plugins/_security/api/tenants/")
+@documentation("Add, delete, or modify multiple tenants in a single call.")
+operation PatchTenants {
+    input: PatchTenants_Input,
+    output: PatchTenants_Output
+}
```

Comment:
Similar issues found in [model/security/get_tenant/operations.smithy](https://github.com/opensearch-project/opensearch-api-specification/pull/113/files#diff-774b3f091e8939f7ab874e3c81bf18409f1d6c13bb002fc529937a68f820e810)